app/controllers/restaurants_controller.rb

```diff
@@ -10,18 +10,18 @@ def index
   # GET /restaurants/1
   # GET /restaurants/1.json
   def show
-    @cuisines= Cuisine.all
+    @cuisines = Cuisine.all
   end
 
   # GET /restaurants/new
   def new
     @restaurant = Restaurant.new
-    @cuisines= Cuisine.all
+    @cuisines = Cuisine.all
```

Comment:
1. why you should do `@cuisines = Cuisine.all` here and in edit?
2. even if you need it you can do `before_action :set_all_cuisine, only [:show, :new, :edit]` instead of calling in each function